Automattic/sensei
# Add Course Completed Actions block

### Changes proposed in this Pull Request
Adds a _Course Completed Actions_ block to pages. The block adds a _Find More Courses_ button that goes to the course archive page when clicked. Additional buttons can be added if desired.

Also adds a `sensei-lms.Course.completedActions` filter to customize the buttons programmatically.

### Testing instructions
- Add `define( 'SENSEI_FEATURE_FLAG_COURSE_COMPLETED_PAGE', true );` to `wp-config.php`.
- Create a new page and add the _Course Completed Actions_ block.
- Add more buttons to the block.
- Preview the page and ensure it goes to the page set as the _Course Archive Page_ when _Find More Courses_ is clicked.
- Go to Courses and Lessons and ensure this block cannot be added there.
- Disable the feature flag and ensure the block is not available on pages and that everything still works / there are no errors.
